### ISSUE TITLE
ERROR: metrics has not been imported

### ISSUE THREAD

**Issue by thisisashwinraj** [venky14/Machine-Learning-with-Iris-Dataset#2]
metrics module has not been imported from sklearn before calculating the accuracy score
`from sklearn.linear_model import LogisticRegression
from sklearn import metrics`
...
...
`model = LogisticRegression()
model.fit(train_X, train_y)
prediction = model.predict(test_X)
print('The accuracy of Logistic Regression is: ', metrics.accuracy_score(prediction, test_y))`
